x-pack/plugins/security_solution/public/common/components/landing_page/onboarding/welcome_header/index.tsx

```diff
@@ -24,7 +24,7 @@ const WelcomeHeaderComponent: React.FC<{ productTier?: ProductTier }> = ({ produ
   const userName = useCurrentUser();
 
   // Full name could be null, user name should always exist
-  const name = userName?.fullName ?? userName?.username;
+  const name = userName?.fullName || userName?.username;
```

Comment:
In this case, the nullish coalescing operator `??` treats only null and undefined as invalid and falls back to the next value, while the logical OR operator `||` considers any falsy value (such as null, undefined, '', 0, false, or NaN) as invalid and falls back to the next value.